tools/openms/.shed.yml

```diff
@@ -1,12 +1,22 @@
-name: openms
-owner: galaxyp
-description: OpenMS Suite for LC/MS data management and analyses
-homepage_url: https://www.openms.de/
-long_description: |
-  OpenMS is an open-source software C++ library for LC/MS data management and analyses. It offers an infrastructure for the rapid development of mass spectrometry related software.
-
-  https://www.openms.de/
-remote_repository_url: https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/openms
-type: unrestricted
-categories:
-- Proteomics
\ No newline at end of file
+name: openms
+owner: galaxyp
+description: OpenMS Suite for LC/MS data management and analyses
+homepage_url: https://www.openms.de/
+long_description: |
+  OpenMS is an open-source software C++ library for LC/MS data management and analyses. It offers an infrastructure for the rapid development of mass spectrometry related software.
+
+  https://www.openms.de/
+remote_repository_url: https://github.com/galaxyproteomics/tools-galaxyp/tree/master/tools/openms
+type: unrestricted
+categories:
+- Proteomics
+auto_tool_repositories:
+  name_template: "{{ tool_id }}"
```

Comment:
They are always unique, so we don't need to take care about this. But this ID appears in log files and so on ... as more information is included as better. But ok, let's leave it like this. Just mention that we could change the generator to include this prefix.